en/material-system/effect-2.x-to-3.0.md

```diff
@@ -0,0 +1,254 @@
+3. Macro options defined in Effect: these macros control whether certain code can be enabled or not (corresponding to pass by pass).
+4. the list of properties defined in Effect (dynamically opened according to the macro definition) and the use of different input boxes to accommodate different types of properties. The editable properties of the editor are generally the mapping of a uniform in the shader, and from v3.0 onwards it is also possible to specify the mapping of a component (using the target parameter in Effect).
+5. v3.0 also added the PipelineStates option, which is mainly used to define the pipeline states of material dependencies, such as DepthStencilState, BlendState, CullMode, etc.
+
+### 1.3 Editor Experience
+
+There are also some differences between v2.x and v3.0 in the editor experience of the material system.
+
+In v3.0, after selecting the node containing the model and material in the **Hierarchy** panel, the **Inspector** panel will display the component properties and the detailed Material configuration panel as follows.
+
+![effect](material-v3.png)
+
+In v2.x, when the node containing the model and material is selected in the **Node Tree** panel, the **Properties** panel will only show the component properties, not the detailed Material configuration panel:
+
+![effect](material-v2x.png)
+
+Instead, you need to jump to the **Assets** panel and select the Material asset before you can edit it in the **Properties** panel:
+
+![effect](material-panel-v2x.png)
+
+## 2. Effect Assets
+
+This section describes the commonalities and differences between assets in v2.x and v3.0.
+
+### 2.1 Effect format and content
+
+For the Effect asset, both v2.x and v3.0 use the YAML1.2 standard syntax and parser, and there are few differences between the two versions. The Effect asset defines a material type, and by following the [syntax format](yaml-101.md), the following information can be defined:
+
+- Technique list of rendering techniques
+- A list of passes for each Technique
+- A list of editable properties exposed to the editor in each Pass (including data type designation within the editor, component mapping relationships, etc.)
+- Shader programs for each Pass, including vertex and slice shader programs
+
+In terms of syntax details, such as Property declarations and macro definitions, the approach is the same:
+
+![effect](material4.jpg)
+
+![effect](material4.1.jpg)
+
+![effect](material5.jpg)
+
+### 2.2 Preset Material Types
+
+There is a big difference between v2.x and v3.0 in terms of preset materials.
+
+- The preset materials in v2.x include 2D Sprite, classic `blinn-phong` lighting materials, `unlit` materials, default `toon` materials, particle materials, etc.
+
+- The preset materials in v3.0 are physics-based rendering system, including standard PBR materials based on physical lighting, Skybox, toon rendering materials, 3D particle materials (CPU & GPU), particle trailing materials, traditional 2D Sprite materials, etc.
```

Comment:
The preset materials in v3.0 are based on physically based rendering, including standard PBR material, Skybox, cartoon style material, ...